examples/gocean/eg5/Makefile

```diff
@@ -110,7 +110,7 @@ $(NAME).%: $(KERNELS) psy_%.o alg.o
 alg.f90 psy.f90: transform
 
 transform: test.x90
-	${PSYCLONE} --profile invokes -l -api "gocean1.0" \
+	${PSYCLONE} --profile invokes -l all -api "gocean1.0" \
```

Comment:
Again, is -l needed? If so mention its use in the READMEs, other remoave. Note the README above this directory does not appear to have entries for eg5 and eg6.

Reply:
Nope, not needed. I've updated the README in the dir above.

Reply:
Great. The README needed the copyright date updating but I've just done that rather than requesting you change it.